### ISSUE TITLE
Hapcut2 caclulate_haplotype_statistics only for single chromosome

### ISSUE THREAD

**Issue by pontushojer** [FrickTobias/BLR#179]
I got the following error when running on a full dataset:

```
ERROR: Chromosome in reference haplotype VCF doesn't match chromosome in VCF used for phasing
reference haplotype VCF: /proj/uppstore2018173/private/analysis/191128.SingleMoleculePhasing/HG002_GRCh38_GIAB_v3.3.2_highconf_triophased.vcf
chr1 != chr2
```

Reading the [docs for the tool](https://github.com/vibansal/HapCUT2/blob/master/utilities/calculate_haplotype_statistics.py) I found that the input VCF can only contain a single chromosome. 

>Calculate statistics on haplotypes assembled using HapCUT2 or similar tools.
Error rates for an assembled haplotype (specified by -v1 and optionally -h1 arguments)
are computed with respect to a "reference" haplotype (specified by -v2 and optionally -h2 arguments).
All files must contain information for one chromosome only!
To compute aggregate statistics across multiple chromosomes, provide files for
each chromosome/contig as an ordered list, using the same chromosome order between flags.
Note: Triallelic variants are supported, but variants with more than 2 alternative alleles
are currently NOT supported. These variants are ignored. Also, variants where the ref and alt
alleles differ between the test haplotype and reference haplotype are skipped.

Working on fix.

**Comment by FrickTobias** [comment]
It's quite common for people to use [hap.py](https://github.com/Illumina/hap.py) for calculating phasing stats. With that said I'm not sure if they calculate the same metrics.

**Comment by pontushojer** [comment]
I started testing hap.py, thank for the tip. 

However I think that this is still quite a nice statistics so I don't just want to abandon this tool. I have two possible solutions for this.

## 1) Split VCF per chromosome.
This is what i tried in #180

Pros:
- Don't need to modify and maintain calculate_haplotype_statistics script

Cons:
- Current solution quite messy 
- Requires folder structure (see discussion in #155 )
- Need to modify "ground truth" phased VCF. 


## 2) Modify calculate_haplotype_statistics to work on current VCFs.
I have made a branch using this approach called [stats-script](https://github.com/FrickTobias/BLR/tree/stats-script) which you can checkout. 

Pros:
- Don't need to modify VCF files. 

Cons:
- Need to maintain complicated script. 

I am not sure which solution is best though. Any input?

**Comment by FrickTobias** [comment]
We went with solution number two, implemented here #181.
